plugins/catalog-backend/src/database/DatabaseManager.ts

```diff
@@ -87,7 +86,7 @@ export class DatabaseManager {
           }
 
           try {
-            const entity = await parser.parse(readerItem.data);
+            const entity = await entityPolicy.apply(readerItem.data);
```

Comment:
Might already be in master, but it is really confusing to have the method be called apply. Reading it you don't know whether `entityPolicy` is a function that we want to pass an explicit this context to, or if it's just a method. I'd call it something like `invoke` or `enforce`. It's also a bit weird that it's a policy that produces some output? I think parser was a bit better in that regard.

Reply:
Well it's also for mutation - a policy can e.g. be a converter from v1beta1 to v1 format if that's how you want to play it, or one that patches up some rare faulty descriptors, or inserts a certain annotation always, or...

Naming is hard, but applying a policy to an object I feel is better than those two suggestions

Reply:
Something like `.use()` then? I feel using `.apply()` is off limits because it does something completely different than the well known method in JavaScript.

Reply:
Oh THAT was what you meant, as in Function.prototype.apply - yeah ok then I understand. I'll see about picking some other name.

Reply:
chose enforce